libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/CodegenLayout.kt

```diff
@@ -115,7 +119,7 @@ internal abstract class CodegenLayout(
   internal fun fragmentVariablesAdapterName(name: String) = fragmentName(name) + "_VariablesAdapter"
   internal fun fragmentSelectionsName(name: String) = regularIdentifier(name) + "Selections"
 
-  internal fun inputObjectName(name: String) = className(name)
+  fun inputObjectName(name: String) = className(name)
```

Comment:
Same side effect as above. Will revisit to see if we can keep `CodegenLayout` internal